Shim to reverse patch part that upstream will patch in lts patchset

diff --git a/./drivers/input/touchscreen/cyttsp5.c.orig b/./drivers/input/touchscreen/cyttsp5.c
index d8715c0f1c..f2a9e70163 100644
--- a/./drivers/input/touchscreen/cyttsp5.c.orig
+++ b/./drivers/input/touchscreen/cyttsp5.c
@@ -631,7 +631,7 @@ static int cyttsp5_power_control(struct cyttsp5 *ts, bool on)
 	int rc;
 
 	SET_CMD_REPORT_TYPE(cmd[0], 0);
-	SET_CMD_REPORT_ID(cmd[0], state);
+	SET_CMD_REPORT_ID(cmd[0], HID_POWER_SLEEP);
 	SET_CMD_OPCODE(cmd[1], HID_CMD_SET_POWER);
 
 	rc = cyttsp5_write(ts, HID_COMMAND_REG, cmd, sizeof(cmd));
